addons/website_sale_slides/views/website_sale_templates.xml

```diff
@@ -16,23 +16,25 @@
     </div>
     <div class="mt-2">
         <t t-foreach="order.order_line" t-as="line">
-            <t t-foreach="line.product_id.channel_ids" t-as="course">
-                <table class="table pb-32 border">
-                    <tr>
-                        <td class="w-50">
-                            <span t-if="course.image_1920" class="w-75" t-field="course.image_1920" t-options="{'widget': 'image'}"/>
-                            <img t-else="" class="img img-responsive w-75" src="/website_slides/static/src/img/channel-training-default.jpg"/>
-                        </td>
-                        <td>
-                            <h3 t-esc="course.name" class="m-2"/>
-                            <div t-field="course.description_short" class="font-weight-light o_wslides_desc_truncate_3 pb-2 ml-2"/>
-                            <a role="button" class="btn btn-primary ml-2" t-attf-href="/slides/#{slug(course)}">
-                                Start Learning
-                            </a>
-                        </td>
-                    </tr>
-                </table>
-            </t>
+            <div t-foreach="line.product_id.channel_ids" t-as="course" class="row mx-0 my-2 border">
+                <div class="col-5 d-flex justify-content-center my-auto">
+                    <span t-if="course.image_1920" t-out="course.image_1920" t-options="{'widget': 'image', 'class': 'my-2'}"/>
```

Comment:
Sorry, it seems that `t-out` doesn't work well with binary field. You can keep `t-field` for this one.